Libraries/Experimental/SwipeableRow/SwipeableFlatList.js

```diff
@@ -88,31 +90,37 @@ class SwipeableFlatList<ItemT> extends React.Component<Props<ItemT>, State> {
   constructor(props: Props<ItemT>, context: any): void {
     super(props, context);
     this.state = {
-      openRowKey: null,
+      extraData: {
+        openRowKey: null,
+      }
     };
 
     this._shouldBounceFirstRowOnMount = this.props.bounceFirstRowOnMount;
+
+    this.onRefFlatList = this.onRefFlatList.bind(this);
```

Comment:
Why this change?

Reply:
This code works faster than fat arrow.
In render we should avoid create function every time
@tomasreimers

Reply:
@kesha-antonov totally get that -- I meant why .bind(this) in the initializer instead of using class fields (like _onScroll)

Reply:
Ping